dnsimple/registrar.go

```diff
@@ -92,7 +92,35 @@ func (s *RegistrarService) GetDomainPremiumPrice(ctx context.Context, accountID
 	return priceResponse, nil
 }
 
-// DomainRegistration represents the result of a domain renewal call.
+// DomainPrice represents the result of a domain prices call.
+type DomainPrice struct {
+	Domain            string  `json:"domain"`
+	Premium           bool    `json:"premium"`
+	RegistrationPrice float64 `json:"registration_price"`
+	RenewalPrice      float64 `json:"renewal_price"`
+	TransferPrice     float64 `json:"transfer_price"`
+}
+
+// DomainPriceResponse represents a response from an API method that results in a domain price list.
+type DomainPriceResponse struct {
+	Response
+	Data *DomainPrice `json:"data"`
+}
+
+func (s *RegistrarService) GetDomainPrices(ctx context.Context, accountID string, domainName string) (*DomainPriceResponse, error) {
```

Comment:
You are missing the docs here. It breaks the lint.

```
➜  dnsimple-go git:(implement_domain_prices) ✗ golint ./...
dnsimple/registrar.go:110:1: exported method RegistrarService.GetDomainPrices should have comment or be unexported
```

Reply:
@weppos thanks!

shouldn't we add this on CI? thanks

Reply:
Yes, it would be great. Sadly, it's a bit complicated:

1. I tried to add a `Makefile` in the past https://github.com/dnsimple/dnsimple-go/commit/b7733b8c7b6a89d5f79f832f66e7feecfbfd6062,  sadly once you add a `Makefile` Travis will rely entirely on it and you will need a non trivial amount of customization to make the build pass in both module and non-module versions. Things would be easier if we would only use module-aware versions (1.11 or above).
2. `golint` is not enough to break the build. It's a highly recommended standard. We could add it as `golint ./...` in the travis.yml file, but it would cause the build to break if anything is invalid (and right now we have one lint warning). The current warning is actually a false positive due to a naming conflict between the webook package and the Webhook as result of the Webhook API. I am not sure how to ignore this as [golint doesn't have a way to address false positives](https://github.com/golang/lint/issues/263).  We should probably upgrade to `golangci-lint` which instead [does](https://golangci-lint.run/usage/false-positives/).